tfhe/src/high_level_api/integers/oprf.rs

```diff
@@ -1,10 +1,43 @@
 use super::{FheIntId, FheUintId};
 use crate::high_level_api::global_state;
 use crate::high_level_api::keys::InternalServerKey;
-use crate::integer::oprf::SignedRandomizationSpec;
 use crate::{FheInt, FheUint, Seed};
 
 impl<Id: FheUintId> FheUint<Id> {
+    /// Generates an encrypted `num_block` blocks unsigned integer
+    /// taken uniformly in `[0, 2^random_bits_count[` using the given seed
+    /// The encryted value is oblivious to the server
+    /// It can be useful to make server random generation deterministic
```

Comment:
docstring does not match the implem, to check all API levels

Reply:
Fixed